src/redux/checkin.js

```diff
@@ -1,5 +1,6 @@
 import produce from 'immer';
-import { pipe, prop, joinOn, curry } from '../utils';
+import * as R from 'ramda';
```

Comment:
Just import what you need, instead of * for better bundle optimization.

The bundle can tree-shake out everything you don't need if you import by name, instead.